Sources/BraintreeCard/Public/BraintreeCard/BTCardNonce.h

```diff
@@ -1,9 +1,6 @@
-#if __has_include(<Braintree/BraintreeCard.h>)
-#import <Braintree/BraintreeCore.h>
-#else
-#import <BraintreeCore/BraintreeCore.h>
-#endif
+typedef NS_ENUM(NSInteger, BTCardNetwork);
```

Comment:
random q: Was this in a different file initially?

Reply:
Yeah, this is part of core and was converted in PR: https://github.com/braintree/braintree_ios/pull/890. We need to forward declare the enum here to use it for the property further down. Previously, we were importing `Braintree.h` here which imported it as part of the `BTEnums.h` file here: https://github.com/braintree/braintree_ios/blob/master/Sources/BraintreeCore/Public/BraintreeCore/BraintreeCore.h#L16